packages/liveblocks-react-ui/src/styles/index.css

```diff
@@ -519,7 +518,6 @@
       &:where([data-state="open"]) {
         block-size: auto;
         opacity: 1;
-        content-visibility: auto;
```

Comment:
Including `content-visibility` was causing issue with chat container not scrolling all the way to the bottom and if a tool call collapsible was the last message in the chat. @marcbouchenoire I recall you mentioned a need to include this but removing it did not break any animations and functionality - maybe I'm missing something?

Reply:
I can't recall exactly what was the issue without it, I tried your branch with/without the Tailwind reset styles, on all browsers, and I also can't see anything breaking 🙌 I did a Safari/Firefox/reset pass a few weeks ago so maybe that "fixed" the need for `content-visibility`, regardless I feel better without it anyway.